include/quicr/object.h

```diff
@@ -11,6 +11,18 @@
 namespace quicr {
     using Extensions = std::map<uint64_t, std::vector<uint8_t>>;
 
+    /**
+     * @brief Status of object as reported by the publisher
+     */
+    enum struct ObjectStatus : uint8_t
+    {
+        AVAIALABLE = 0x0,
+        DOES_NOT_EXIST = 0x1,
+        END_OF_GROUP = 0x3,
+        END_OF_TRACK = 0x4,
+        END_OF_SUBGROUP = 0x5
```

Comment:
I think naming enums like this goes against the styling.
```suggestion
        kAvailable = 0x0,
        kDoesNotExist = 0x1,
        kEndOfGroup = 0x3,
        kEndOfTrack = 0x4,
        kEndOfSubgroup = 0x5
```